doc/getting_started.rst

```diff
@@ -48,6 +48,7 @@ For an existing app:
        }
 
        dependencies {
+           // Definition for the 3DR Services Client Library
```

Comment:
@hamishwillee let's bump the client library version to `2.7.+`. I have also updated the sample `Hello Drone` app to match.